lib/kitchen/directions/eob_composite_page_container/v1.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Kitchen::Directions::EobCompositePageContainer
+  class V1
+    renderable
+
+    def bake(container_key:, uuid_key:, metadata_source:, content:, append_to:)
+      @title = I18n.t(:"eoc.#{container_key}")
+      @uuid_key = uuid_key
+      @container_class_type = container_key
+      @metadata = metadata_source.children_to_keep.copy
+      @content = content
+      @in_composite_chapter = append_to.is?(:composite_chapter)
```

Comment:
I'm not sure if we need this here :thinking: In EOB only composite chapter is Answer Key. The question is whether we want to use this module there? I didn't work on any Answer Key in Kitchen, so it's hard for me to have any opinion about that, so maybe someone else have any thoughts about advantages and disadvantages of using that in solutions.